lib/features/children/overview/widgets/child_overview_item.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:givt_app/app/routes/routes.dart';
+import 'package:givt_app/core/enums/amplitude_events.dart';
+import 'package:givt_app/features/auth/cubit/auth_cubit.dart';
+import 'package:givt_app/features/children/overview/cubit/children_overview_cubit.dart';
+import 'package:givt_app/features/children/overview/models/profile.dart';
+import 'package:givt_app/utils/utils.dart';
+import 'package:go_router/go_router.dart';
+
+class ChildOverviewItem extends StatelessWidget {
+  const ChildOverviewItem({
+    required this.profile,
+    super.key,
+  });
+
+  final Profile profile;
+  @override
+  Widget build(BuildContext context) {
+    final user = context.read<AuthCubit>().state.user;
+    final currencySymbol = Util.getCurrencySymbol(countryCode: user.country);
+
+    return Container(
+      margin: const EdgeInsets.symmetric(horizontal: 5),
+      child: ElevatedButton(
+        style: ElevatedButton.styleFrom(
+          shadowColor: profile.monsterColorWithAlpha,
+          elevation: 0,
+          padding: const EdgeInsets.symmetric(horizontal: 10, vertical: 20),
+          shape: RoundedRectangleBorder(
+            borderRadius: BorderRadius.circular(20),
+            side: BorderSide(
+              width: 2,
+              color: profile.monsterColor,
+            ),
+          ),
+          backgroundColor: profile.monsterColorWithAlpha,
+        ),
+        onPressed: () {
+          AnalyticsHelper.logEvent(
+              eventName: AmplitudeEvents.childProfileClicked,
+              eventProperties: {
+                'name': profile.firstName,
+              });
+          context.pushNamed(
+            Pages.childDetails.name,
+            extra: [
+              context.read<ChildrenOverviewCubit>(),
+              profile,
+            ],
+          );
+        },
+        child: SizedBox(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              ClipRRect(
+                borderRadius: BorderRadius.circular(10),
+                child: SvgPicture.network(
+                  profile.pictureURL,
+                  width: 70,
+                  height: 70,
```

Comment:
I am going to change this to dynamic sizing, because between iPhone SE and iPhone14 i cannot solve the overflow with this at 70, and the fontsize also being fixed.